pkg/drivers/kic/kic.go

```diff
@@ -134,6 +136,9 @@ func (d *Driver) Create() error {
 		glog.Infof("Starting extracting preloaded images to volume ...")
 		// Extract preloaded images to container
 		if err := oci.ExtractTarballToVolume(d.NodeConfig.OCIBinary, download.TarballPath(d.NodeConfig.KubernetesVersion, d.NodeConfig.ContainerRuntime), params.Name, d.NodeConfig.ImageDigest); err != nil {
+			if strings.Contains(err.Error(), "No space left on device") {
+				exit.Message(reason.RsrcInsufficientDockerStorage, "preload extraction failed: \"No space left on device\"")
```

Comment:
I am not a big fan of libraries to exit, how about retruning the error to the caller ? and ensure the caller fails fast and does not do retry?

Reply:
Sure, done.